README.md

```diff
@@ -22,22 +22,22 @@ Below is an overview of the included packages, with a short description of their
 
 These are the primary dependencies required to use this project.
 
-- ROS 2 [Humble](https://docs.ros.org/en/humble/Installation.html)
-- Gazebo [Fortress](https://gazebosim.org/docs/fortress)
+- ROS 2 [Jazzy](https://docs.ros.org/en/jazzy/Installation.html)
+- Gazebo [Harmonic](https://gazebosim.org/docs/harmonic)
 
-All additional dependencies are either pulled via [vcstool](https://wiki.ros.org/vcstool) ([panda_ign_moveit2.repos](./panda_ign_moveit2.repos)) or installed via [rosdep](https://wiki.ros.org/rosdep) during the building process below.
+All additional dependencies are either pulled via [vcstool](https://wiki.ros.org/vcstool) ([panda_gz_moveit2.repos](./panda_gz_moveit2.repos)) or installed via [rosdep](https://wiki.ros.org/rosdep) during the building process below.
 
 ### Building
 
-Clone this repository, import dependencies, install dependencies and build with [colcon](https://colcon.readthedocs.io).
+Clone this repository, import dependencies, install dependencies, and build with [colcon](https://colcon.readthedocs.io).
 
 ```bash
 # Clone this repository into your favourite ROS 2 workspace
-git clone https://github.com/AndrejOrsula/panda_ign_moveit2.git
+git clone https://github.com/AndrejOrsula/panda_gz_moveit2.git
 # Import dependencies
-vcs import < panda_ign_moveit2/panda_ign_moveit2.repos
+vcs import < panda_gz_moveit2/panda_gz_moveit2.repos
```

Comment:
```suggestion
```
With the changes here I think that we can remove the repos file [panda_ign_moveit2.repos](https://github.com/AndrejOrsula/panda_gz_moveit2/blob/master/panda_ign_moveit2.repos).